habitTrackerApp.py

```diff
@@ -401,16 +425,14 @@ def build(self):
 - this might be useful for getting all the togglebuttons: static get_widgets(groupname) 
     Return a list of the widgets contained in a specific group. If the group doesn’t exist, an empty list will be returned.
 
+    #Bookmark 18 Feb 2022. There is still apparently some issue with currentSideButton
+            #Bookmark 18 Feb 2022. Important functionality missing is that clicking on sidebuttons does not
+            #save the state of the streaks
+
     Note
     Always release the result of this method! Holding a reference to any of these widgets can prevent them from being garbage collected. If in doubt, do:
     l = ToggleButtonBehavior.get_widgets('mygroup')
     # do your job
```

Comment:
Usually best to add all this comments at the top as this is where we start to read :D